src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Data/SqlServer/ApplicationDbContextModelSnapshot.cs

```diff
@@ -16,7 +15,8 @@ protected override void BuildModel(ModelBuilder modelBuilder)
         {
 #pragma warning disable 612, 618
             modelBuilder
-                .HasAnnotation("ProductVersion", "2.1.0-preview1-28193")
+                .HasAnnotation("ProductVersion", "2.1.0-rc1")
```

Comment:
Real last one...

Reply:
Yeah the good news is I just have to update it at the top of updatemigrations.cmd and run it once :)